drv/ArrayFIFOQueue.drv

```diff
@@ -147,6 +147,18 @@ public class ARRAY_FIFO_QUEUE KEY_GENERIC implements PRIORITY_QUEUE KEY_GENERIC,
 		array[--start] = x;
 		if (end == start) expand();
 	}
+	
+	/** Allows read-only access into the queue.
+	 * @param index the index to peek at.
+	 */
+	public KEY_GENERIC_TYPE peek(int index)
+	{
+		if(index >= size()}
```

Comment:
I expect unit tests. This won't even compile.

Reply:
Eh could you explain why?
Also technically this is not really needed?

Reply:
Bracket instead of paren...

Reply:
Yeah ok... Github editing is bad (and me too with it). And I hadn't had the time to actually make a real project out of it. Honestly also because I was scared of making bigger changes since i am new to this type of coding.